migrate-to-github/what-to-migrate.md

```diff
@@ -0,0 +1,73 @@
+## What we want to migrate
+
+#### Git history (obviously)
+
+#### Match IDs (important)
+
+#### Migrate labels
+
+### Issues
+
+- Link to a corresponding archived issue in Pagure
+- Transfer labels
+
+Options to discuss:
+
+- Migrate them as they are in Pagure
+- Shorten them:
+  - squish all the comments in the description of the issue like this to save the API
+    calls:
+
+
+Issue description
+___
+User A: Comment 1
+___
+User B: Comment 2
+___
+etc.
+___
+
+Shortening issues may be useful to save some amount of API calls, because Github has
+a limit of 5000 per hour. The disadvantage is the opacity of the resulting Issue. 
```

Comment:
I wouldn't do any compromises because of GitHub limits. We will run the script under a bot user, so we shouldn't get our accounts banned/blocked/or something. And since we know the limits, we can easily batch the requests to comply with them. Also, we IMHO don't care if the script runs 10 seconds or 10 days.